docs/api.md

```diff
@@ -367,6 +367,33 @@ documented in the [Browser API ⇗](/docs/browser.md) documentation.
 
 * See [Browser API ⇗](/docs/browser.md)
 
+#### `transport` (Object)
+
+The `transport` option is a shortcut to the [pino.transport()](#pino-transport) function.
+It supports the same input options:
+```js
+require('pino')({
+  transport: {
+    target: '/absolute/path/to/my-transport.mjs'
+  }
+})
+
+// or multiple transports
+require('pino')({
+  transport: {
+    targets: [
+      { target: '/absolute/path/to/my-transport.mjs', level: 'error' },
+      { target: 'some-file-transport', options: { destination: '/dev/null' }
+    ]
+  }
+})
+```
+
+It is not possible to provide an additional [`destination`](#destination) argument
```

Comment:
this is confusing, and also in the error message we call it stream, instead of destination argument. 

```suggestion
If the transport option is supplied to `pino`, a [`destination`](#destination) parameter may not also be passed as a separate argument to `pino`:

```js
pino({ transport: {}}, '/path/to/somewhere') // THIS WILL NOT WORK, DO NOT DO THIS
pino({ transport: {}}, process.stderr) // THIS WILL NOT WORK, DO NOT DO THIS
```